torch/utils/cpp_extension.py

```diff
@@ -828,6 +828,35 @@ def CUDAExtension(name, sources, *args, **kwargs):
                 cmdclass={
                     'build_ext': BuildExtension
                 })
+
+    Compute capabilities:
+
+    By default the extension will be compiled to run on all archs of the cards visible during the
+    building process of the extension, plus PTX. If down the road a new card is installed the
+    extension may need to be recompiled.
+
+    If a visible card has a compute capability (CC) that's newer than the newest version for which
+    your nvcc can build fully-compiled binaries, Pytorch will make nvcc fall back to building
+    kernels with the newest version of PTX your nvcc does support (see below for details on PTX).
+
+    However, you can explicitly specify which archs you want the extension to support like so:
```

Comment:
I think we should make it clearer this is an alternative to the defaults.
"
You can override the default behavior using `TORCH_CUDA_ARCH_LIST` to explicitly specify which archs you want the extension to support:
"

Reply:
done